react/components/LoginLogout.tsx

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable no-console */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+import { LoginAccountDialog } from 'extensions.account-dialog'
+import React, { Fragment, useEffect, useState } from 'react'
+import { useCssHandles } from 'vtex.css-handles'
```

Comment:
Nitpick: put React imports on top. It just looks a little bit cleaner

```suggestion
import React, { Fragment, useEffect, useState } from 'react'

import { LoginAccountDialog } from 'extensions.account-dialog'
import { useCssHandles } from 'vtex.css-handles'
```

Reply:
Our lints are configured to order imports in alphabetical order and no line between them: 
![image](https://user-images.githubusercontent.com/12221113/98024943-cf72f300-1de7-11eb-8e67-a9b1f1144e5d.png)

Should we change this configuration?

Reply:
Hmmmm I see. Keep the way it is, then